tests/operator/suite.go

```diff
@@ -126,6 +126,53 @@ func LoadChecks() {
 			testOperatorCatalogSourceBundleCount(c, &env)
 			return nil
 		}))
+
+	checksGroup.Add(checksdb.NewCheck(identifiers.GetTestIDAndLabels(identifiers.TestOperatorInstallationInTenantNamespace)).
+		WithSkipCheckFn(testhelper.GetNoOperatorsSkipFn(&env)).
+		WithCheckFn(func(c *checksdb.Check) error {
+			testOperatorInstallationInTenantNamespace(c, &env)
+			return nil
+		}))
+}
+
+// This function checks if SingleNamespaced Operators should only be installed in the tenant dedicated operator namespace
+func testOperatorInstallationInTenantNamespace(check *checksdb.Check, env *provider.TestEnvironment) {
+	check.LogInfo("Starting testInstalledSingleNamespaceOperatorInTenanttNamespace")
+
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	singleNamespacedOperators := filterSingleNamespacedOperatorUnderTest(env.Operators)
+
+	// namespaces to be considered
+	var namespaces []string
+	for i := range singleNamespacedOperators {
+		operator := singleNamespacedOperators[i]
+		namespaces = append(namespaces, operator.Namespace)
```

Comment:
The list of namespaces under test should be the list of namespaces in which operator under test are installed. If an operator is clusterwide and  installed in the namespace, it should be part of the non-compliant logs (see: if operator.IsClusterWide && operator.Csv.Name == foundCsv)

Reply:
I do not think so, this test case considers only operators under test which are installed with single name space. So, may be we need a new skip method. Wdyt?

Reply:
@sebrandon1

Reply:
I'm okay with a specific test for 3rd party operators so we can be more explicit about which 3rd party operators are outside of `openshift-operators` namespace.